### PR TITLE
Feat: create member manager model

### PR DESCRIPTION
**Pull request by TanNguyenEH**
# Pivotal tracker (URL):

# Summary

# Changes proposed in this pull request:
-
-
-

### CODE REVIEW

File: app/models/member_manager.rb [hunk 1]
```diff
@@ -0,0 +1,10 @@
+class MemberManager < Sequel::Model(DB[:member_managers])
+  many_to_one :manager, class: :Member
+  many_to_one :member
+
+  def validate
+    super
+    validates_presence %i[manager_id member_id]
```
Comment (hieubui149) [inline app/models/member_manager.rb]: `[manager_id, member_id]` should be unique as well, right?

File: db/migrations/20200320112647_create_member_managers.rb [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+Sequel.migration do
+  up do
+    create_table :member_managers do
+      primary_key :id, :uuid, default: Sequel.function(:uuid_generate_v4)
+      foreign_key :manager_id, :members, type: :uuid, null: false
+      foreign_key :member_id, :members, type: :uuid, null: false
+      column :level, Integer
+      column :primary_manager, Integer
+      unique %i[member_id level]
```
Comment (hieubui149) [inline db/migrations/20200320112647_create_member_managers.rb]: [member_id manager_id] should be unique here as well

File: app/models/member.rb [hunk 3]
```diff
@@ -2,6 +2,8 @@ class Member < Sequel::Model(DB[:members])
   many_to_one :organisation
   many_to_many :groups
   many_to_many :teams
+  one_to_many :managers, class: :MemberManager, key: :manager_id
+  one_to_many :members, class: :MemberManager, key: :member_id
```
Comment (hieubui149) [inline app/models/member.rb]: I think we don't need it. `member.members` returns all records that I'm a members and `member.managers` returns all records that I'm a managers. It makes no sense

Comment (TanNguyenEH) [reply app/models/member.rb]: yes, I think so but before I didn't sure about that so I still added it. Now, I gonna remove it. Thank you.

File: app/models/member_manager.rb (hunk 1 above)
Comment (TanNguyenEH) [reply app/models/member_manager.rb]: you're right, let me update it

**Pull request merged**